package.json

```diff
@@ -28,5 +28,8 @@
     "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check --exclude 'examples/**/*' --exclude '**/node_modules/**' --exclude '**/*.d.ts' '**/*.ts'",
     "test": "mocha --opts test/mocha.opts.root 'packages/*/test/**/*.ts'",
     "posttest": "npm run lint"
+  },
+  "dependencies": {
+    "reflect-metadata": "^0.1.10"
```

Comment:
I agree with @rmg, this dependency must go to `packages/authentication.package.json`. I have already pointed that out in my earlier https://github.com/strongloop/loopback-next/pull/224#discussion_r113707386, with a link to a different place where we are already doing the same: https://github.com/strongloop/loopback-next/blob/a422dfb699f512f2daf2692b5cf56395c2425c5b/packages/context/package.json#L15